.github/workflows/dockerhub.yml

```diff
@@ -0,0 +1,71 @@
+name: dockerhub
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+env:
+  G_PULL_REQUEST_BRANCH: ${{ github.head_ref }}
+  G_BRANCH: ${{ github.base_ref }}
+  G_PR_REPO_SLUG: ${{ github.event.pull_request.head.repo.full_name }}
+  G_REPO_SLUG: ${{ github.repository }}
+  G_PULL_REQUEST: ${{ github.event.number }}
+  G_REF: ${{ github.ref }}
+  EVENT: ${{ github.event_name }}
+  BUILD_TAG: ''
+  IS_FORK: ''
```

Comment:
You do not need that since you have the condition `if: github.repository == 'FORTH-ICS-INSPIRE/artemis-web'`